validphys2/src/validphys/process_options.py

```diff
@@ -252,6 +252,7 @@ def _displusjet_xq2map(kin_dict):
     "HQP_PTQ": HQP_PTQ,
     "HERAJET": HERAJET,
     "HERADIJET": dataclasses.replace(HERAJET, name="HERADIJET", description="DIS + jj production"),
+    "DY_Z_Y" : dataclasses.replace(DY_W_ETA, name="DY_Z_Y", description="DY Z -> ll rapidity")
```

Comment:
I made use of the process type implemented by MC, but I changed name and description. Moreover, note that the accepted kinematic variable in `DY_W_ETA` is `eta`. Should I use `y` instead, since LHCb data is in the boson rapidity?

Reply:
I think both are ok. Actually, we could call it also DY_RAP or something like that and unify both with a common description.

But changing these (internal) names are "as easy" as a search & replace.

Reply:
Should I rebase on top of master again? Currently `DY_W_ETA` is not in `process_options`.

Reply:
Yes, if you need to use it yes.